test/test_ops.py

```diff
@@ -388,6 +466,101 @@ def test_vmapvjp(self, device, dtype, op):
             for loop_out, batched_out in get_fallback_and_vmap_exhaustive(fn, args, {}):
                 self.assertEqual(loop_out, batched_out, atol=1e-4, rtol=1e-4)
 
+    # There are several variations we care about
+    # 1) primal batched (TODO)
+    # 2) tangent batched (batched grads) <--
+    # 3) both batched (TODO)
+    # The below tests (2) only.
+    @ops(functorch_lagging_op_db + additional_op_db, allowed_dtypes=(torch.float,))
+    @skipOps('TestOperators', 'test_vmapjvp', {
+        # These should to be fixed:
+
+        # Random failures: Copied from above
+        xfail('nn.functional.dropout'),  # randomness
+        xfail('double'),
+        xfail('linalg.householder_product'),
+
+        # Batching rule not implemented for aten::_local_scalar_dense
+        xfail('linalg.eigh'),
+        xfail('linalg.tensorinv'),
+        xfail('linalg.inv'),
+        xfail('linalg.matrix_power'),
+        xfail('linalg.cholesky'),
+
+        # Batching rule not implemented for aten::logical_{and,or}.out
+        xfail('cumprod'),
+        xfail('__rpow__'),
+        xfail('clamp', 'scalar'),
+        xfail('logit'),
+        xfail('fmin'),
+        xfail('fmax'),
+        xfail('float_power'),
+        xfail('pow'),
+
+        # Try to in-place batched tensor into non-batched tensor
+        xfail('diag_embed'),
+        xfail('nn.functional.pad', 'circular'),
+        xfail('matrix_exp'),
+        xfail('lu'),
+        xfail('fill_'),
+        xfail('block_diag'),
+        xfail('index_add'),
+        xfail('index_copy'),
+        xfail('index_put'),
+        xfail('index_fill'),
+        xfail('masked_fill'),
+        xfail('masked_scatter'),
+
+        # Expected the output of forward differentiable view operations
+        # to have the tangent have the same layout as primal
+        xfail('linalg.multi_dot'),
+        xfail('einsum'),
+        xfail('reshape'),  # this is odd
```

Comment:
I can't repro these failures -- are they CUDA-only? (i have a cpu-only build)

Reply:
This is on CPU, you aren't able to replicate it because I tested this on top of the stack in core! See https://github.com/pytorch/pytorch/pull/66293

Reply:
Ah I see this now. I was not checking out your stack in core correct, thanks for the correction!